<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoDataBaseDialog</class>
 <widget class="QDialog" name="NoDataBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>There is no data base!</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Saaghar can not open the database file, Saaghar needs its database for working properly. Choose one of the following options.</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="exitPushButton">
        <property name="text">
         <string>Exit!</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="selectDataBase">
        <property name="toolTip">
         <string>&lt;p&gt;By selecting this option opens a dialog, and you should select a path containing 'ganjoor.s3db' file. Then Saaghar starts by setting it as its default data base!&lt;/p&gt;</string>
        </property>
        <property name="whatsThis">
         <string>By selecting this option opens a dialog, and you should select a path containing 'ganjoor.s3db' file. Then Saaghar starts by setting it as its default data base!</string>
        </property>
        <property name="text">
         <string>Select new path for  data base</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="description">
         <string>Select folder containing 'ganjoor.s3db' file!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="createDataBaseFromLocal">
        <property name="toolTip">
         <string>&lt;p&gt;Firstly a dialog appears for select a path for saving 'ganjoor.s3db' file. It has not to contain 'ganjoor.s3db'! And then a dialog appears that you should select the data sets you want to install.&lt;/p&gt;</string>
        </property>
        <property name="whatsThis">
         <string>Firstly a dialog appears for select a path for saving 'ganjoor.s3db' file. It has not to contain 'ganjoor.s3db'! And then a dialog appears that you should select the data sets you want to install.</string>
        </property>
        <property name="text">
         <string>Create data base and install existing data sets to it.</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="description">
         <string>Select folder for saving 'ganjoor.s3db' file! And then select the data sets you want to install.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="createDataBaseFromRemote">
        <property name="toolTip">
         <string>&lt;p&gt;Firstly a dialog appears for select a path for saving 'ganjoor.s3db' file. It has not to contain 'ganjoor.s3db'! And then a dialog appears that you should select the data sets you want to download and install.&lt;/p&gt;</string>
        </property>
        <property name="whatsThis">
         <string>Firstly a dialog appears for select a path for saving 'ganjoor.s3db' file. It has not to contain 'ganjoor.s3db'! And then a dialog appears that you should select the data sets you want to download and install.</string>
        </property>
        <property name="text">
         <string>Create data base then download and install data sets to it.</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="description">
         <string>Select folder for saving 'ganjoor.s3db' file! And then select the data sets you want to download and install.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Error:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Data Base Path:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
